### PR TITLE
My Portfolio

### PR DESCRIPTION
**Pull request by audukehinde**
# My Portfolio Mobile Skeleton

### In the project:

* I converted a figma design into a webpage
* I added github flow
* I added linters
* I added flexbox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# My Porfolio
```
Comment (SajjadAhmad14) [inline README.md:1]: Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

File: index.html [hunk 2]
```diff
@@ -4,9 +4,77 @@
...
+  <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
   <title>My Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <a class="wrap logo" href="#">Kehinde Audu</a>
+      <i class="fas fa-bars"></i>
```
Comment (SajjadAhmad14) [inline index.html:22]: Please use the `nav` tag for the hamburger menu only.

File: index.html [hunk 3]
```diff
@@ -4,9 +4,77 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
   <title>My Portfolio</title>
 </head>
 <body>
-  
+  <header>
```
Comment (SajjadAhmad14) [inline index.html:19]: Please follow the font-family for the header from the design page.

File: index.html [hunk 4]
```diff
@@ -4,9 +4,77 @@
...
+      <div class="abilities">
+        <div class="languages wrap">
+          <p>Languages</p>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
```
Comment (SajjadAhmad14) [inline index.html:42]: Please use `section` as a container for the Headline section.

File: index.html [hunk 5]
```diff
@@ -4,9 +4,77 @@
...
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div>
+      <img class="elipse" src="img/elipse.svg.svg" alt="">
```
Comment (SajjadAhmad14) [inline index.html:66]: Please do not use the `img` tag for decoration images. They should be background images.

File: index.html [hunk 6]
```diff
@@ -4,9 +4,77 @@
...
+        <div class="languages wrap">
+          <p>Languages</p>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="frameworks wrap">
```
Comment (SajjadAhmad14) [inline index.html:43]: Please use font-family for the Headline section from the design page.

File: index.html [hunk 7]
```diff
@@ -4,9 +4,77 @@
...
+        <div class="skills wrap">
+          <p>Skills</p>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
```
Comment (SajjadAhmad14) [inline index.html:62]: Please use font-family and color for `Languages`, `Skills`, and `Frameworks` from the design page.

File: index.html [hunk 8]
```diff
@@ -4,9 +4,77 @@
...
+      </div>
+    </div>
+    <div>
+      <img class="elipse" src="img/elipse.svg.svg" alt="">
+    </div>
+    <div class="socials">
+      <i class="fab fa-github"></i>
+      <i class="fab fa-linkedin-in"></i>
+      <i class="fab fa-twitter"></i>
+      <i class="fab fa-medium-m"></i>
```
Comment (SajjadAhmad14) [inline index.html:72]: The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kenny Audu, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (audukehinde) [reply README.md:1]: Hello Sajjad, please I'm new to this, could you explain what I need to do. I'm some how confused. Thanks for your understanding

File: index.html (hunk 2 above)
Comment (audukehinde) [reply index.html:22]: Hello Sajjad, please I'm new to this, could you explain what I need to do. I'm some how confused. Thanks for your understanding

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!! #
Hi @audukehinde, 

Good job implementing the changes requested by the previews reviewer!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**